Sources/ApolloTestSupport/TestMock.swift

```diff
@@ -67,6 +67,7 @@ public class Mock<O: MockObject>: AnyMock, Hashable {
     }
   }
 
+  @_disfavoredOverload
```

Comment:
This is required to resolve a build error `Ambiguous use of '_set(_:for:)'` when the initializer has both single value and array values.